pkg/storaged/details.jsx

```diff
@@ -43,14 +43,19 @@ export class StdDetailsLayout extends React.Component {
                         { this.props.alert }
                         { this.props.header }
                     </GridItem>
-                    <GridItem md={8} lg={9}>
-                        <div id="detail-content">
-                            { this.props.content }
-                        </div>
-                        <JobsPanel client={this.props.client} />
-                    </GridItem>
-                    <GridItem id="detail-sidebar" md={4} lg={3}>
-                        { this.props.sidebar }
+                    <GridItem span={12}>
```

Comment:
This Grid wrapper is not needed anymore then. Removing it should have 0 effect.

Reply:
The whole Grid still has an effect, it produces the gutter between header and content.  It might be a bit overkill for that now, though...